pkg/service/common.go

```diff
@@ -47,3 +47,9 @@ func SanitizeDataNode(datanode *InfoNode_DataNode) {
 func SanitizeAnalyzerNode(analyzerNode *Analyzer) {
 	analyzerNode.NodeType = NodeTypeAnalyzerNode
 }
+
+func SanitizePackageNode(packageNode *PackageNode, uid string) {
+	packageNode.NodeType = NodeTypePackageNode
+	packageNode.Session = ""
```

Comment:
Is this working? Seems like you are overriding the session in the database with an empty string.